crates/debug/src/transform/expression.rs

```diff
@@ -462,76 +528,244 @@ impl<'a, 'b> ValueLabelRangesBuilder<'a, 'b> {
         }
         self.processed_labels.insert(label);
 
-        let value_ranges = if let Some(frame_info) = self.frame_info {
-            &frame_info.value_ranges
-        } else {
-            return;
+        let value_ranges = match self.frame_info.and_then(|fi| fi.value_ranges.get(&label)) {
+            Some(value_ranges) => value_ranges,
+            None => {
+                return;
+            }
```

Comment:
```suggestion
        let value_ranges = self.frame_info
	    .and_then(|fi| fi.value_ranges.get(&label))
	    .unwrap_or({return});
```
This might be a little more clear. Matter of taste.

Reply:
It is probably evil. Sorry for this :-/
The only reference a search turned up was https://www.reddit.com/r/rust/comments/86x1fj/unwrap_orreturn/
and I am now convinced, that an eagerly evaluated argument `return` when passed to `unwrap_or` will exit the function in any case. I will learn a bit more about this antipattern, but don't apply it 💀!